packages/dashboards/addon/controllers/dashboards/dashboard/view.js

```diff
@@ -0,0 +1,116 @@
+/**
+ * Copyright 2019, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ */
+
+import Controller from '@ember/controller';
+import { inject as service } from '@ember/service';
+import { isArray } from '@ember/array';
```

Comment:
Can we use this instead: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/isArray